Tests/SentryTests/Protocol/SentrySdkInfoTests.swift

```diff
@@ -0,0 +1,107 @@
+import XCTest
+
+class SentrySdkInfoTests: XCTestCase {
+    
+    private let sdkName = "sentry.cocoa"
+
+    func testWithPatchLevelSuffix() {
+        let actual = SentrySdkInfo(sdkName: sdkName, andVersionString: "50.10.20-beta1")
+        
+        XCTAssertEqual(sdkName, actual.sdkName)
+        XCTAssertEqual(50, actual.versionMajor)
+        XCTAssertEqual(10, actual.versionMinor)
+        XCTAssertEqual(20, actual.versionPatchLevel)
+    }
+    
+    func testWithGarbagePatchLevelSuffix() {
+        let actual = SentrySdkInfo(sdkName: sdkName, andVersionString: "50.10.20a101a")
```

Comment:
M: on Android, this would be 50.10 only, it's either a number or nothing, we can discuss that to see which one makes more sense, IMO, it's either a number or not set.

Reply:
Thanks for pointing this out. With changing to https://develop.sentry.dev/sdk/event-payloads/sdk/ this comment gets irrelevant.